Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -226,6 +227,15 @@ static std::optional<CSSWideKeyword> consumeCSSWideKeyword(CSSParserTokenRange&
     return keyword;
 }
 
+// MARK: - function value consumer
+
+static bool consumeCSSFunctionValue(CSSParserTokenRange& range, CSSPropertyID property, CSS::PropertyParserState& state, CSS::PropertyParserResult& result)
+{
+    if (range.peek().functionId() == CSSValueInternalAutoBase)
```

Comment:
I think you'll need to gate this to `state.context.mode == UASheetMode` to avoid web-exposing this.